### ISSUE TITLE
How to re-use analysis

### ISSUE THREAD

**Issue by scottepeterson** [sjurgis/yachtworld-puppeteer#2]
Great work @sjurgis .  And the analysis looks great too.  Any help on how to insert a new dataset and reuse the analysis sheet easily?  I dropped in the results to the datasheet (2nd tab)... no luck on that.

**Comment by sjurgis** [comment]
Hi @scottepeterson that should work.
Just upload the new data onto second sheet (yachtworld-1582369882131).
When importing, under **Import location** choose _Replace current sheet_.

That said, I've just checked and it does break some of the additional analysis I've added recently (formula fields like `price / feet-year`). So perhaps it's best to wipe existing rows and use _Replace data at selected cell_.

Worth noting I've tried to analyse change of prices over time (one of the sheets there), but starting to think spreadsheet is not the best option here...
